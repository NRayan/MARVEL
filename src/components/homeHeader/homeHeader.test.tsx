import { render } from "@testing-library/react-native";
import React from "react";
import { RenderWithNavigation, RenderWithTheme } from "../../tests";
import { HomeHeader } from "./";

const Component = () => (<HomeHeader searchEnabled />);

describe("HomeHeader", () => {

	it("Should Render correctly", () => {

		const tree = render(
			<RenderWithTheme>
				<RenderWithNavigation component={Component} />
			</RenderWithTheme>
		).toJSON();

		expect(tree).toMatchSnapshot();
	});
});