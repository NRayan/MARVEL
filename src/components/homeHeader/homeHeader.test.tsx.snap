// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeHeader Should Render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNSScreenStack
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RNSScreen
      forwardedRef={[Function]}
      onAppear={[Function]}
      onDisappear={[Function]}
      onDismissed={[Function]}
      onTransitionProgress={[Function]}
      onWillDisappear={[Function]}
      replaceAnimation="push"
      stackPresentation="push"
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <RNSScreenStackHeaderConfig
        backButtonInCustomView={false}
        backgroundColor="rgb(255, 255, 255)"
        color="rgb(0, 122, 255)"
        direction="ltr"
        disableBackButtonMenu={false}
        hidden={false}
        hideBackButton={false}
        largeTitleHideShadow={false}
        title="MockedScreen"
        titleColor="rgb(28, 28, 30)"
        topInsetEnabled={false}
        translucent={false}
      />
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            Object {
              "backgroundColor": "rgb(242, 242, 242)",
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "space-between",
                "marginBottom": 10,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#FFF",
                  "fontFamily": "Inter_700Bold",
                  "fontSize": 22,
                },
              ]
            }
          >
            MARVEL
          </Text>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text />
          </View>
        </View>
      </View>
    </RNSScreen>
  </RNSScreenStack>
</RNCSafeAreaProvider>
`;
